kiwi/repository/base.py

```diff
@@ -64,7 +64,7 @@ def runtime_config(self):
     def add_repo(
         self, name, uri, repo_type, prio, dist, components,
         user, secret, credentials_file, repo_gpgcheck, pkg_gpgcheck,
-        sourcetype
+        sourcetype, use_for_debootstrap
```

Comment:
Regarding the interface character of the API I think that parameter should be generalized, e.g ```use_for_bootstrap=True|False```

Reply:
I am fine with it, change done. I just added it without the default at base class level as it is the case for the other parameters, but I agree that would be good to also include the default at the base class, for clarity and to keep API consistent.